portal/templates/assignments/edit-assignments.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'layouts/base.html' %}
```

Comment:
you seem to be re-adding several files that were moved over to the new file structure. All of the duplicates need to be deleted and the ones in the new structure should be used instead.